pkg/translator/jaeger/traces_to_jaegerproto.go

```diff
@@ -80,10 +74,7 @@ func resourceSpansToJaegerProto(rs pdata.ResourceSpans) (*model.Batch, error) {
 		spans := ils.Spans()
 		for j := 0; j < spans.Len(); j++ {
 			span := spans.At(j)
-			jSpan, err := spanToJaegerProto(span, ils.InstrumentationLibrary())
-			if err != nil {
-				return nil, err
-			}
+			jSpan := spanToJaegerProto(span, ils.InstrumentationLibrary())
 			if jSpan != nil {
```

Comment:
Having a nil span being returned in case of an invalid ID would cause a single span to be ignored.

Reply:
I don't think the conversion/translation should do "validation". We can have a processor or other library to do these kind of validation.

Also we don't do this when receiving (converting from Jaeger to OTLP) data, why when exporting?